samples/dashboard/demos/dashboard-strava-csv/demo.css

```diff
@@ -0,0 +1,7 @@
+@import "https://code.highcharts.com/css/highcharts.css";
+@import "https://code.highcharts.com/css/datagrid.css";
+@import "https://code.highcharts.com/css/dashboard/gui.css";
+
+#csv {
```

Comment:
I prefer to use class instead of ID. 

Suggestion: 
- create a class `.hidden` or add `style="display: none;" like in other Highcharts demo is